packages/react-chat/src/views/index.ts

```diff
@@ -1,3 +1,3 @@
+export { default as ChatEmbed } from './ChatEmbed';
```

Comment:
moved this to a top level view - it is technical equivalent to `ChatWidget` (which we should rename to `ChatOverlay`, but it might break people using it as a component or cause massive diffs with forks)